workflows/general/alignment-post.wdl

```diff
@@ -8,6 +8,7 @@ import "https://raw.githubusercontent.com/stjude/XenoCP/4.0.0-alpha/wdl/workflow
 
 workflow alignment_post {
     meta {
+        name: "Alignment Post"
```

Comment:
I'm not sure I'm a fan of having to specify these since they're mostly a rewritten form of the workflow name in the WDL.

Reply:
Compromise, we can probably auto-create a `name` in `wdl-doc` by taking the workflow name and "title capitalizing" it. Then we leave out the `name` field for most of our workflows and only specify when it's a special case (like the ones that use parentheses).

Reply:
That compromise sounds sane to me, but I also really don't think it's a big deal to expect users to type this field for each workflow. So I could go either way.

Reply:
I think that's probably reasonable (though you could omit it for now). Split the name on `_` characters and capitalize the resulting words, if `meta.name` does not exist.

Reply:
I forget the crate name, but the crate we use for the `SnakeCase` and `PascalCase` rules should make this pretty trivial. It's a nifty crate, I was impressed when we first added it.